src/sass/chrome.scss

```diff
@@ -57,6 +57,12 @@ aside {
     background: var(--pf-c-page__main-section--BackgroundColor);
 }
 
+// NoAccess styling
+#no-access {
```

Comment:
Two things here:

1. Can we do `.className#no-access` selector instead of just the ID? (where className is the class name for this ID)
2. Can we split up the height and background and then attach it to the `// Page layout styles` & the `// Page root background color`, respectively?